src/components/CustomerQuota/NoQuota/TransactionTitle.tsx

```diff
@@ -7,32 +7,24 @@ import {
   formatTimeDifference,
 } from "../../../utils/dateTimeFormatter";
 import { useTranslate } from "../../../hooks/useTranslate/useTranslate";
-import { useTheme } from "../../../context/theme";
-import { GOVWALLET_THEME_NAME } from "../../../common/styles/themes";
 
 export const DistantTransactionTitle: FunctionComponent<{
   transactionTime: Date;
   toggleTimeSensitiveTitle: boolean;
 }> = ({ transactionTime, toggleTimeSensitiveTitle }) => {
-  const { theme } = useTheme();
-  const { i18nt } = useTranslate();
+  const { c13nt, i18nt } = useTranslate();
+  const limitReachedDateKey = "limitReachedDate";
   const today = toggleTimeSensitiveTitle
     ? `${i18nt("checkoutSuccessScreen", "today")}`
     : "";
   return (
     <>
       <AppText style={sharedStyles.statusTitle}>
-        {theme.name === GOVWALLET_THEME_NAME
-          ? i18nt(
-              "checkoutSuccessScreen",
-              "previouslyRecordedDate",
-              undefined,
-              {
-                dateTime: formatDateTime(transactionTime),
-                today,
-              }
-            )
-          : i18nt("checkoutSuccessScreen", "limitReachedDate", undefined, {
+        {c13nt(limitReachedDateKey) !== limitReachedDateKey
+          ? c13nt(limitReachedDateKey)
+              .replace("${dateTime}", formatDateTime(transactionTime))
+              .replace("${today}", today)
```

Comment:
Not sure if this is the best approach, wanted to use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) (i.e. `${dateTime}` and assign the value) but wanted to keep it consistent with the rest of the text, hence using this format instead. Do comment if there are ways this can be done better!